lib/lrud.js

```diff
@@ -75,8 +75,8 @@ const getParentContainer = (elem) => {
 const getFocusables = (scope) => {
   if (!scope) return [];
 
-  const ignoredElements = toArray(scope.querySelectorAll('.' + ignoredClass));
-  if (scope.className.indexOf(ignoredClass) > -1) ignoredElements.push(scope);
+  const ignoredElements = toArray(scope.querySelectorAll(ignoreSelector));
+  if (scope.className.indexOf(ignoreSelector) > -1) ignoredElements.push(scope);
```

Comment:
```suggestion
```
Is this line still necessary? This is expecting that the `className` of the element you pass would match the entire string of `ignoreSelector`.